features/list_search.feature

```diff
@@ -0,0 +1,95 @@
+Feature: List Search
+    
+@Basic
+Scenario: 
+    Then I install the ccz app at "test_list_search.ccz"
+    Then I login with username "test_list_search" and password "123"
+    Then I scroll until I see the "Start" text
+    Then I touch the "Start" text    
```

Comment:
Hm, do we think that is more robust? Honestly "Seeing text on the screen and clicking it" feels less prone to change than "very specifically encoding how that item is represented in code as of today and how to choose it"